src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsteriskInvalid.java

```diff
@@ -0,0 +1,62 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+
+
+
+/**Some Javadoc with violation.
+ *
+ */
+class InputJavadocMissingWhitespaceAfterAsteriskInvalid
+{
+    /**
+     *<p>Some Javadoc with violation.
+     */
+    class InnerInputJavadocMissingWhitespaceAfterAsteriskInValid {
+
+        /**
+         *Some Javadoc with violation.
+         */
+        public static final byte NUL = 0;
+
+        /**
+         * Some Javadoc.
+         *
+         *<p>Some Javadoc with violation.
+         */
+        void bar1() {}
+
+        /***Some Javadoc with violation. ***/
+        void bar2() {}
+    }
+
+    /****
+     * Some Javadoc.
+     *@see Something with violation
+     ****/
+    void foo1() {}
+
+    /**
+     *Some Javadoc with violation.
+     **/
+    public static final int foo2 = 0;
+
+    /**
+     *Some Javadoc with violation.
+     */
+    enum Foo3 {}
+
+    /**
+     *<pre>
+     *   Some Javadoc with violation.
+     *</pre>
```

Comment:
please place word violation on each line where violation is present.
it should be clear where exactly violation be placed , without reading of UT expectations.

in other words: each line at `final String[] expected = {` should have word `violation` in input file, on same line.

Reply:
done and report updated